public/team.json

```
@@ -188,6 +188,18 @@
     "email": "alexren",
     "website": "",
     "avatar": "https://cachet.dunkirk.sh/users/U06PR6B8D37/r"
+  },
+   {
+    "name": "Euan Ripper",
+    "department": "HQ",
+    "role": "Engineering Gap year YSWS",
```

```suggestion
    "role": "Engineering",
```
For consistency I'd keep this as Engineering similar to the rest of the gap years then include that you are a gap year in your bio if u choose.

---

yeah seems good ty!